src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println("Truck started its work.");

```

Comment:
The message 'Truck started its work.' is not very informative. Consider specifying that it is a Truck that is working, to make use of the informative messages as mentioned in the checklist.